haas/ext/obm/haas_obm.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2013-2015 Massachusetts Open Cloud Contributors
```

Comment:
Are we doing recursive HaaS with this submit, or just OBM drivertization? If the latter, then this should be removed.